internal/provider/resource_monitor.go

```diff
@@ -200,6 +200,26 @@ var monitorSchema = map[string]*schema.Schema{
 		Type:        schema.TypeString,
 		Optional:    true,
 	},
+	"request_headers": {
+		Description: "An array of request headers, consisting of name and value pairs",
+		Type:        schema.TypeList,
```

Comment:
@adikus 💄  do you want to keep the whitespace here? Seems a bit odd to have it here but not in front of the rest of the values